src/Stratis.Bitcoin.Features.Consensus/CoinViews/CoinView.cs

```diff
@@ -50,7 +51,7 @@ public abstract class CoinView
         /// If the i-th item of <see cref="FetchCoinsResponse.UnspentOutputs"/> is <c>null</c>, it means that there are no unspent outputs in the given transaction.
         /// </para>
         /// </returns>
-        public abstract Task<FetchCoinsResponse> FetchCoinsAsync(uint256[] txIds);
+        public abstract Task<FetchCoinsResponse> FetchCoinsAsync(uint256[] txIds, CancellationToken ct = default(CancellationToken));
```

Comment:
would be nice if you added parameter comment if you are adding parameter to a commented method

Reply:
Honestly I don't see any relevant comment I can put here :(

Reply:
See above